invenio_app_rdm/config.py

```diff
@@ -661,3 +661,7 @@ def files_rest_permission_factory(obj, action):
     'sort': ['bestmatch', 'newest', 'oldest'],
 }
 """User requests search configuration (i.e list of user requests)"""
+
+RDM_REQUESTS_ROUTES = {
+    'user-dashboard-community-submission': '/me/requests/<pid_value>',
```

Comment:
I am not sure this is about the dashboard (search). Isn't it about the request details page?
```suggestion
    'user-requests-community-submission': '/me/requests/<pid_value>',
```

Reply:
as far as I understood, there will be different endpoints for different ways of reaching the request details page, this is only one of them (user dashboard)
another one is for example community requests (which will end up as something similar to `/<community>/requests/<pid>`
Ping @lnielsen

Reply:
Correct